ajax/libs/qcobjects/package.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "qcobjects",
+  "version": "2.1.298",
+  "description": "QCObjects a Quickcorp Objects Library - Cross Browser Javascript Framework for MVC Patterns",
+  "main": "qcobjects.min.js",
```

```suggestion
  "main": "QCObjects.js",
```
This file needs to exist in the npm package, it appears only the unminfiied version `QCObjects.js` does.

---

QCObjects doesn't need to be minified as the dynamic packages loading model is meant to warranty split and one-demand loading of scripts resources. If it is an explicit requirement for cdnjs I could minify the main file QCObjects.js for compatibility purposes. In the meantime, I've just changed the name to point to the right one (QCObjects.js). Cheers.